### ISSUE TITLE
Setting up Eslint and making the code prettier

### ISSUE THREAD

**Issue by subhangi2731** [ADRE9/bunk-manager-mern#154]
I will lint the code and set up Eslint in the project

**Comment by subhangi2731** [comment]
@ADRE9 @kunaljain0212 @Mustafiz04 please asign this issue to me

**Comment by kunaljain0212** [comment]
@subhangi2731 would you be able to work on two issues at once? I would suggest you to work on #144 at the moment and once that is completed we'll assign this to you.

**Comment by subhangi2731** [comment]
@kunaljain0212 yes i can work on this please assign I have experience in eslint and prettier

**Comment by subhangi2731** [comment]
I will send a pr today only @kunaljain0212

**Comment by subhangi2731** [comment]
@ADRE9 please assign me so that I can start working

**Comment by subhangi2731** [comment]
okay @ADRE9 @kunaljain0212 I will open this issue once I complete the other issue

**Comment by ADRE9** [comment]
Take your time. But be in touch with mentors about your progress.

**Comment by subhangi2731** [comment]
sure @ADRE9 I will be in touch

**Comment by kunaljain0212** [comment]
Any updates? @subhangi2731

**Comment by subhangi2731** [comment]
@kunaljain0212 I am working on it
